spec/lib/tic_tac_toe_spec.rb

```diff
@@ -44,11 +35,31 @@
   end
 
   context 'when start is called' do
-    it 'should call display_welcome and display_board' do
+    it 'should display welcome, instructions,the board, and prompt a move' do
       expect(tic_tac_toe).to receive(:display_welcome)
```

Comment:
Will this test ensure that you receive the welcome message and then the other methods will be executed in the order you expect?